ci.sh

```diff
@@ -3,35 +3,40 @@
 set -e
 set -o pipefail
 
+# add npm packages to $PATH
+export PATH=$(pwd)/node_modules/.bin:$PATH
+
 # set up code coverage instrumentation
-rm -rf coverage
-export npm_config_coverage=true
+rm -rf coverage .nyc_output
 
-npm test
+# run unit tests
+tap --reporter dot --coverage --no-coverage-report test/js/*/*.js
 
-# allow writing core files
+# allow writing core files for render tests
 ulimit -c unlimited -S
 echo 'ulimit -c: '`ulimit -c`
 echo '/proc/sys/kernel/core_pattern: '`cat /proc/sys/kernel/core_pattern`
 sysctl kernel.core_pattern
 
-npm run test-suite
-RESULT=$?
+# run render tests
+istanbul cover --dir .nyc_output --include-pid --report none --print none test/render.test.js &&
+istanbul cover --dir .nyc_output --include-pid --report none --print none test/query.test.js
```

Comment:
`nyc` is a wrapper around `istanbul`

The `nyc` CLI doesn't support our use case of running multiple test suites and aggregating their results. We'd have to do more file-copying or folder-renaming gymnastics to make it work.